recipes/boost/all/conanfile.py

```diff
@@ -1078,8 +1086,16 @@ def add_defines(library):
             flags.append("define=BOOST_ERROR_CODE_HEADER_ONLY=1")
         if self.options.system_no_deprecated:
             flags.append("define=BOOST_SYSTEM_NO_DEPRECATED=1")
+        if self.options.asio_disable_eventfd:
+            flags.append("define=BOOST_ASIO_DISABLE_EVENTFD=1")
+        if self.options.asio_disable_serial_port:
+            flags.append("define=BOOST_ASIO_DISABLE_SERIAL_PORT=1")
         if self.options.asio_no_deprecated:
             flags.append("define=BOOST_ASIO_NO_DEPRECATED=1")
+        if self.options.asio_no_experimental_string_view:
+            flags.append("define=BOOST_ASIO_DISABLE_STD_EXPERIMENTAL_STRING_VIEW=1")
+        if self.options.beast_use_std_string_view:
+            flags.append("define=BOOST_BEAST_USE_STD_STRING_VIEW=1")
```

Comment:
My concern here is these change really quickly and managing them with each version is a pain https://github.com/boostorg/beast/commit/7300ef463bbc702ac5659f42fa3b8b528ae575e3

This one is already deprecated 🙊